src/blocks/toolbox.js

```diff
@@ -0,0 +1,135 @@
+
+export const toolbox = {
```

Comment:
Can we use the normal toolbox and just add another category for the P5 blocks?

Reply:
I would like to stick to a non-category toolbox while we try to figure out how to do basic navigation within a stack and within a workspace. I think the workflow for categories in toolboxes is fairly standard and adding it in just adds complexity to the basic test scenarios.

I'm targeting the simple use case (up to ten blocks on a workspace, up to three stacks, and no categories in the toolbox) for the intro sequences that will use this first.